site/docs/manual/Architecture.md

```diff
@@ -55,6 +48,15 @@ By default your application should have at least one `world`. A world is basical
 world = new World();
 ```
 
+The `World` constructor accept an option object with the following parameters:
+- ***entityClass***: Provide the base class for entities that implements or extends `Entity`.
+- ***entityPoolSize***: Define the initial entity pool size for entities. It can help to avoid GC during execution if the application keep extending the pool dynamically at execution time.
```

Comment:
```suggestion
- ***entityPoolSize***: Define the initial entity pool size for entities. It can help to avoid GC during execution if the application expands the pool dynamically at execution time.
```